registry.k8s.io/README.md

```diff
@@ -124,3 +124,17 @@ requests, which is described in detail
 [post-promo-job]: https://testgrid.k8s.io/sig-release-releng-blocking#post-k8sio-image-promo
 [ci-promo-job]: https://testgrid.k8s.io/sig-release-releng-blocking#ci-k8sio-image-promo
 [project-github]: https://git.k8s.io/community/github-management#project-owned-organizations
+
+### Publishing helm charts on registry.k8s.io
+
+Publishing a helm chart in the Kubernetes organization follows a process similar to
+publishing an image.
+
+1. The subproject should build, document, and support the helm chart development.
+2. The subproject should push the chart to the staging docker repo.
+3. On release tags, one can promote the chart and images to registry.k8s.io by obtaining the sha
+   for the tagged helm chart and the tagged image from the build logs,
+   which should be linked.
+4. Open a PR into your images directory (registry.k8s.io/images/k8s-staging-SUBPROJECT/images.yaml)
+5. Add chart and image sha to the respective sections.
+6. Once PR is merged, confirm both images and helm charts exist by `helm install subproject oci://registry.k8s.io/subproject/charts/subproject --version=$VERSION`
```

Comment:
If there any way to trigger the `helm-chart-push` manually? I think the helm chart (in lws) generated with the tag release is not working as expected, I need to build a new helm chart.

Reply:
I don't think this is a way to trigger these jobs manually. Am I correct in that @BenTheElder?